data/boosters/dmu-box-topper

```diff
@@ -0,0 +1,9 @@
+# Data from https://magic.wizards.com/en/news/feature/dominaria-united-product-overview-2022-08-18
```

Comment:
You left this extra file without extension.

Reply:
Oops, I'll remove that.